src/test/descriptor_tests.cpp

```diff
@@ -297,29 +302,39 @@ void DoCheck(const std::string& prv, const std::string& pub, const std::string&
     BOOST_CHECK_MESSAGE(left_paths.empty(), "Not all expected key paths found: " + prv);
 }
 
-void Check(const std::string& prv, const std::string& pub, const std::string& norm_prv, const std::string& norm_pub, int flags, const std::vector<std::vector<std::string>>& scripts, const std::optional<OutputType>& type, const std::set<std::vector<uint32_t>>& paths = ONLY_EMPTY)
+void Check(const std::string& prv, const std::string& pub, const std::string& norm_prv, const std::string& norm_pub,
+           int flags, const std::vector<std::vector<std::string>>& scripts, const std::optional<OutputType>& type,
+           const std::set<std::vector<uint32_t>>& paths = ONLY_EMPTY, uint32_t spender_nlocktime=0, uint32_t spender_nsequence=0)
```

Comment:
In 96e9de9304edf5feec87adf5ee56ad0f526459b8 "script/sign: signing support for Miniscript with timelocks"

In `DoCheck`, `spender_nsequence` is `CTxIn::SEQUENCE_FINAL`, should it be that here as well (instead of 0)?

Reply:
Yeah, whichever works. I'm putting `CTxIn::SEQUENCE_FINAL`.